gateway/src/project.rs

```diff
@@ -849,9 +849,8 @@ pub mod exec {
 
     use super::*;
 
-    pub async fn revive(gateway: GatewayService) -> Result<(), ProjectError> {
+    pub async fn revive(gateway: Arc<GatewayService>) -> Result<(), ProjectError> {
```

Comment:
Probably better to send this down the global work queue that is used by other API endpoints as well, instead of creating our own separately.

For that, grab `sender` here and use `TaskBuilder::send`.